pkg/metrics/metrics.go

```diff
@@ -739,7 +739,7 @@ type LegacyMetrics struct {
 
 func NewLegacyMetrics() *LegacyMetrics {
 	lm := &LegacyMetrics{
-		BootstrapTimes: metric.NewHistogramVec(metric.HistogramOpts{
+		BootstrapTimes: metric.NewGaugeVec(metric.GaugeOpts{
```

Comment:
Note: This might be a good candidate for `NewGaugeVecWithLabels` as the labels appear to be static

Reply:
Yep, that's definitely possible. I haven't gone that route at this moment as it would essentially lead to ~ double the cardinality (30 vs ~15) due to registering the `outcome` label, which in the vast majority of the cases is always `success`. Happy to configure the labels if you feel that approach to be better though.